File: Codewars/Python/Splitting_strings.py
def solution(s):
    if len(s) % 2 == 1:
        s += '_'
    some_array = []

    for i in range (len(s) // 2):
        some_array.append(s[0:2])
        s = s[2::]
    
    return some_array

File: Codewars/CPP/Splitting_strings.cpp
#include <string>
#include <vector>
#include <iostream>

std::vector<std::string> solution(const std::string &s)
{
    std::vector <std::string> kekw;
    std::string somestring = s;
    if (somestring.length() % 2 == 1)
    {
        somestring += '_';
    }
    
    while (!somestring.empty())
    {
        kekw.push_back(somestring.substr(0, 2));
        somestring.erase(0, 2);
    }
    
    return kekw;
}

int main () {
    std::vector<std::string> soluted = solution("aabbc");
    for (std::string s : soluted)
    {
        std::cout << s << ' ';
    }
    return 0;
}